daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -560,7 +568,8 @@ class RazerBlackWidowUltimate2016(_RippleKeyboard):
     """
     Class for the Razer BlackWidow Ultimate 2016
     """
-    EVENT_FILE_REGEX = re.compile(r'.*BlackWidow_Ultimate_2016(-if01)?-event-kbd')
+    EVENT_FILE_REGEX = re.compile(
+        r'.*BlackWidow_Ultimate_2016(-if01)?-event-kbd')
```

Comment:
Please remove all the reformats that you didn't touch manually. (You maybe didn't use the max-line-length or whatever option with autopep8?)